templates/squad_adversarial/AddSent/templates.yaml

```diff
@@ -0,0 +1,74 @@
+dataset: squad_adversarial
+subset: AddSent
+templates:
+  0b69eb58-0ad7-45b4-baf2-f3abac775fe2: !Template
+    id: 0b69eb58-0ad7-45b4-baf2-f3abac775fe2
+    jinja: 'Generate a title for the following context-question pair:
```

Comment:
I think the title only really applies to the context - so you could either remove the question from this prompt, or create a separate prompt where the question doesn't appear.